definitions/family-private/json/AuthorisationCaseEvent.json

```diff
@@ -33,5 +33,40 @@
     "CaseEventID": "solicitorCreate",
     "UserRole": "caseworker-privatelaw-superuser",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/07/2021",
```

Comment:
I know this field is ignored, but could we use current date please? When it comes to releasing it was helping a bit (especially when we had feature toggles)